src/scvi/nn/_base_components.py

```diff
@@ -15,7 +15,13 @@ def _identity(x):
 
 
 class FCLayers(nn.Module):
-    """A helper class to build fully-connected layers for a neural network.
+    """FCLayers class of scvi-tools adapted to also inject continous covariates.
+
+    The only adaptation is addition of `n_cont` parameter in init and `cont` in forward,
```

Comment:
Please remove this docstring part  - I only added it as info for implementing the change.